spec/package.json

```diff
@@ -10,6 +10,7 @@
     "chai-as-promised": "^7.1.1",
     "coffee-script": "1.12.7",
     "dbus-native": "^0.2.3",
+    "fs-extra": "^6.0.0",
```

Comment:
Is this needed?

Reply:
Gah, no that's from an unrelated task that shouldn't've been in this PR. Good catch; will remove